src/sinks/databend/service.rs

```diff
@@ -0,0 +1,217 @@
+use bytes::Bytes;
```

Comment:
Can we move this into the second group of third party crates?

Reply:
fixed